nlohmann_json.podspec

```diff
@@ -0,0 +1,89 @@
+#
+#  Be sure to run `pod spec lint nlohmann_json.podspec' to ensure this is a
+#  valid spec and to remove all comments including this before submitting the spec.
+#
+#  To learn more about Podspec attributes see http://docs.cocoapods.org/specification.html
+#  To see working Podspecs in the CocoaPods repo see https://github.com/CocoaPods/Specs/
+#
+
+Pod::Spec.new do |s|
+
+  # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  s.name         = "nlohmann_json"
+  s.version      = "3.1.2"
+  s.summary      = "JSON reader & writer for C++ optimized for ease of use and modern C++14"
+
+
+  # CocoaPods strips indentation for description
+  s.description  = <<-DESC
+    JSON library for c++, it can read & write, uses C++14 features to have
+    modern syntax.
+                   DESC
+
+  s.homepage     = "https://github.com/nlohmann/json"
+  s.screenshots  = "https://raw.githubusercontent.com/nlohmann/json/master/doc/json.gif"
+
+
+  # ―――  Spec License  ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  Licensing your code is important. See http://choosealicense.com for more info.
+  #  CocoaPods will detect a license file if there is a named LICENSE*
+  #  Popular ones are 'MIT', 'BSD' and 'Apache License, Version 2.0'.
+  #
+
+  license_text = File.read("LICENSE.MIT")
+  # cocoapods checks extension and expects it to be a txt. so instead
+  # we read the file and pass it in as text.
+  s.license      = { :type => "MIT", :text => license_text }
+
+
+  # ――― Author Metadata  ――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  Specify the authors of the library, with email addresses. Email addresses
+  #  of the authors are extracted from the SCM log. E.g. $ git log. CocoaPods also
+  #  accepts just a name if you'd rather not provide an email address.
+  #
+  #  Specify a social_media_url where others can refer to, for example a twitter
+  #  profile URL.
+  #
+
+
+  s.authors            = {
+    "Niels Lohmann" => "nosend@respectmyprivacy.com"
+  }
+
+  # ――― Platform Specifics ――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  If this Pod runs only on iOS or OS X, then specify the platform and
+  #  the deployment target. You can optionally include the target after the platform.
+  #
+
+  s.platform     = :ios, "8.0"
```

Comment:
macOS should also be fine, right?

Reply:
I only tested iOS, theoretically it should be fine on macOS, watch and all other platforms. I don't want to add untested platforms.